Automattic/jetpack
# Social | Add globalize/unglobalize UI and logic

## Proposed changes:

* Adds a button to the connections table to allow marking and unmarking a connection as shared.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Ensure that feature flag is ON - `define( 'JETPACK_SOCIAL_USE_ADMIN_UI_V1', true );`
- Test the following for both Jetpack and Social plugin
    - Goto settings page
    - Add some connections
    - Confirm that you see the _"Mark as shared"_ button
    - Click on it
    - Confirm that the API request is a success
    - Confirm that the label changes to _"Mark as not shared"_
   

https://github.com/Automattic/jetpack/assets/18226415/fa83de89-6995-4b99-9c1b-bfa15f757110
